src/app/users/[id]/page.tsx

```
@@ -0,0 +1,73 @@
+ */
+const UserPage = async (
+  // Next.js13のApp Routerでは、引数のparamsに、
+  // []で囲まれたフォルダ名のidが動的ルーティングの情報として渡ってくる
+  { params }: { params: { id: string }}
+) => {
+  // URLパラメータからユーザーIDを数値として取得
+  const userId = Number(params.id);
+  // ユーザーデータを非同期で取得
+  const userData = await getUserData(userId);
+  // 取得したデータを分割代入
+  const { user, products } = userData;
```

ESLint

[error] 'products' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)